flipperdevices/flipperzero-firmware
# [FL-1824] Dolphin refactoring

# What's new

- Refactoring
- Separate dolphin service and desktop app
- Access favorite app by pressing `Left` key on main standby screen
- Desktop settings added, currently only for configuring favorite app


# Verification 

- Flash
- Check setting up favorite app from settings
- Verify all other stuff works (Locking/unlocking, etc)

# Checklist (do not modify)
